test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/ProjectSystems/LegacyPackageReferenceProjectTests.cs

```diff
@@ -849,6 +851,397 @@ public async Task GetPackageSpecAsync_CentralPackageVersionsRemovedDuplicates()
             Assert.Equal(VersionRange.Parse(packageB.Version), centralPackageVersions[packageB.PackageId].VersionRange);
         }
 
+        [Fact]
+        public async Task GetInstalledVersion_WithAssetsFile_ReturnsVersionsFromAssetsSpecs()
```

Comment:
I did not dig too much in these tests, the names seem identical to `NetCorePackageReference` which is how they should be. 

So I trust you are doing the right thing here :)